policies/assembler.md

```diff
@@ -0,0 +1,19 @@
+Policy for Accepting Assembler Optimisations
+============================================
+
+New assembler optimisations for any algorithm having a C based implementation in
+any provider are always acceptable (subject to standard review procedures) for
+all platforms in the master branch.
+
+New assembler optimisations are never acceptable for any platform or provider in
+a stable release branch.
+
+Updates to existing assembler optimistations in a stable release branch are only
+acceptable where such updates would be allowed under the ["Stable Release
+Updates" policy][1].
+
+Where assembler optimisations are acceptable they should be implemented using
+[perlasm][2].
```

Comment:
Why don't you just say: "Assembler optimisations should be implemented using [perlasm][2]."

Reply:
We could do. But I don't think the current wording is wrong.